narwhals/_pyspark/expr.py

```diff
@@ -0,0 +1,213 @@
+            returns_scalar=self._returns_scalar or returns_scalar,
+            backend_version=self._backend_version,
+            dtypes=self._dtypes,
+        )
+
+    def __add__(self, other: PySparkExpr) -> Self:
+        return self._from_call(operator.add, "__add__", other, returns_scalar=False)
+
+    def __sub__(self, other: PySparkExpr) -> Self:
+        return self._from_call(operator.sub, "__sub__", other, returns_scalar=False)
+
+    def __mul__(self, other: PySparkExpr) -> Self:
+        return self._from_call(operator.mul, "__mul__", other, returns_scalar=False)
+
+    def __lt__(self, other: PySparkExpr) -> Self:
+        return self._from_call(operator.lt, "__lt__", other, returns_scalar=False)
+
+    def __gt__(self, other: PySparkExpr) -> Self:
+        return self._from_call(operator.gt, "__gt__", other, returns_scalar=False)
+
+    def alias(self, name: str) -> Self:
+        def _alias(df: PySparkLazyFrame) -> list[Column]:
+            return [col.alias(name) for col in self._call(df)]
+
+        # Define this one manually, so that we can
+        # override `output_names` and not increase depth
+        return self.__class__(
+            _alias,
+            depth=self._depth,
+            function_name=self._function_name,
+            root_names=self._root_names,
+            output_names=[name],
+            returns_scalar=self._returns_scalar,
+            backend_version=self._backend_version,
+            dtypes=self._dtypes,
+        )
+
+    def count(self) -> Self:
+        def _count(_input: Column) -> Column:
+            from pyspark.sql import functions as F  # noqa: N812
+
+            return F.count(_input)
+
+        return self._from_call(_count, "count", returns_scalar=True)
+
+    def max(self) -> Self:
+        def _max(_input: Column) -> Column:
+            from pyspark.sql import functions as F  # noqa: N812
+
+            return F.max(_input)
+
+        return self._from_call(_max, "max", returns_scalar=True)
+
+    def mean(self) -> Self:
+        def _mean(_input: Column) -> Column:
+            from pyspark.sql import functions as F  # noqa: N812
+
+            return F.mean(_input)
+
+        return self._from_call(_mean, "mean", returns_scalar=True)
+
+    def min(self) -> Self:
+        def _min(_input: Column) -> Column:
+            from pyspark.sql import functions as F  # noqa: N812
+
+            return F.min(_input)
+
+        return self._from_call(_min, "min", returns_scalar=True)
+
+    def std(self, ddof: int = 1) -> Self:
+        import numpy as np  # ignore-banned-import
+
+        def _std(_input: Column) -> Column:  # pragma: no cover
+            if self._backend_version < (3, 4) or parse_version(np.__version__) > (2, 0):
+                from pyspark.sql.functions import stddev
+
+                _ = ddof
+                return stddev(_input)
+            from pyspark.pandas.spark.functions import stddev
+
+            return stddev(_input, ddof=ddof)
```

Comment:
Not sure this is ideal. Unfortunately the `stddev` in pyspark SQL does not support `ddof`
From `3.4` they added one function in the pandas namespace that supports that.

Based on which version one has, `std` may return a different result :( Any ideas?

Reply:
thanks :)